File: app/src/main/java/mk/android/com/canvasdrawview/presenter/CanvasTouch.java
package mk.android.com.canvasdrawview.presenter;

import android.view.MotionEvent;

/**
 * Created by Mayuri Khinvasara on 02,December,2018
 */
public interface CanvasTouch {
     void onClickEvent(MotionEvent event);
     void onLongPressEvent(float initialTouchX, float initialTouchY);
}

File: app/src/main/java/mk/android/com/canvasdrawview/presenter/CanvasTouch.kt
package mk.android.com.canvasdrawview.presenter

import android.view.MotionEvent

/**
 * Created by Mayuri Khinvasara on 02,December,2018
 */
/*
     Interface to detect canvas touch and canvas long press events
 */
interface CanvasTouch {
    fun onClickEvent(event: MotionEvent)
    fun onLongPressEvent(initialTouchX: Float, initialTouchY: Float)
}
